Source/WebCore/loader/SubresourceLoader.cpp

```diff
@@ -294,9 +295,9 @@ void SubresourceLoader::willSendRequestInternal(ResourceRequest&& newRequest, co
         }
 
         auto originalOrigin = SecurityOrigin::create(redirectResponse.url());
-        m_site = m_documentLoader->cachedResourceLoader().computeFetchMetadataSite(newRequest, m_resource->type(), options().mode, originalOrigin, m_site);
+        m_site = protectedDocumentLoader()->cachedResourceLoader().computeFetchMetadataSite(newRequest, m_resource->type(), options().mode, originalOrigin, m_site);
```

Comment:
Seems we should cache the DocumentLoader in a RefPtr local variable since it is used at least twice. Avoids refunding churn.

Reply:
Fixed.